readme.md

```diff
@@ -0,0 +1 @@
+Test file
```

Comment:
This file could be written as markdown

Reply:
Thanks, I'll change that. Fixed in newest commit.